dist/profile/templates/confluence/vhost.conf

```
@@ -2857,6 +2857,9 @@ RewriteRule "^/display/JENKINS/App\+Center\+Plugin$" "https://plugins.jenkins.io
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/XML\+Job\+to\+Job\+DSL$" "https://plugins.jenkins.io/xml-job-to-job-dsl/" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Task\+Scanner\+Plugin" "https://plugins.jenkins.io/warnings-ng//" [NC,L,QSA,R=301]
```

No need for the extra trailing slash
```suggestion
RewriteRule "^/display/JENKINS/Task\+Scanner\+Plugin" "https://plugins.jenkins.io/warnings-ng/" [NC,L,QSA,R=301]
```